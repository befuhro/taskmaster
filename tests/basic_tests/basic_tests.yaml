programs:
  basic_tests:
    cmd: "./basic_tests"
    numprocs: 1
    umask: 022
    workingdir: /tmp
    autostart: true
    autorestart: unexpected
    exitcodes:
      - 0
      - 2
    startretries: 3
    starttime: 0
    stopsignal: "terminated"
    stoptime: 10
#    stdout: ./basic_tests.stdout
#    stderr: ./basic_tests.stderr
    env:
      STARTED_BY: taskmaster
      ANSWER: 42