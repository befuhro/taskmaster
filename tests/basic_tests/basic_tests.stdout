44967 started at: 19:13:42.17771
